Lets Do This/Controllers/Campaign/LDTCampaignDetailViewController.m

```diff
@@ -329,6 +343,22 @@ - (CGSize)collectionView:(UICollectionView *)collectionView layout:(UICollection
 
     if (indexPath.section == LDTCampaignDetailSectionTypeCampaign) {
         if (indexPath.row == LDTCampaignDetailCampaignSectionRowCampaign) {
+            // Pitch view (user hasn't signed up or completed):
+            if (![[self user] isDoingCampaign:self.campaign] && ![[self user] hasCompletedCampaign:self.campaign]) {
+                UINib *campaignCellNib = [UINib nibWithNibName:@"LDTCampaignListCampaignCell" bundle:nil];
+                LDTCampaignListCampaignCell *sizingCell =  [[campaignCellNib instantiateWithOwner:nil options:nil] firstObject];
+                [self configureCampaignPitchCell:sizingCell];
+                sizingCell.frame = CGRectMake(0, 0, CGRectGetWidth(self.collectionView.bounds), CGRectGetHeight(sizingCell.frame));
+                NSLog(@"sizingCell.frame.height %f", CGRectGetHeight(sizingCell.frame));
+                [sizingCell setNeedsLayout];
+                [sizingCell layoutIfNeeded];
+                // This is returning 0
+//                CGFloat campaignCellHeight = [sizingCell.contentView systemLayoutSizeFittingSize:UILayoutFittingCompressedSize].height;
```

Comment:
This isn't working because the `UIView` labelled `Action Container View` in the `LDTCampaignListCampaignCell.xib` is missing a bottom constraint to the `UICollectionViewCell` superview. However, adding this breaks the `LDTCampaignListCampaignViewController` -- which is what I will continue working on in a new branch after merging this.